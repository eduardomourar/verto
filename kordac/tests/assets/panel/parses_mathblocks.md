{panel type="extra-for-experts" summary="Mathblocks"}
It's hard to write that down (although you could use “”), so we have a symbol especially for it, which is the Greek letter epsilon: {math}\epsilon{math end}.
You'll come across {math}\epsilon{math end} quite often with formal languages.
{panel end}