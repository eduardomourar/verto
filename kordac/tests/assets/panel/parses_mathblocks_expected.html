<div class="clearfix">
<ul class="collapsible panel" data-collapsible="accordion">
<li class="panel-selector panel-extra-for-experts">
<div class="collapsible-header">
<div class="panel-heading">Extra For Experts: Mathblocks</div>
<div class="dropdown-menu-arrow">▼</div>
</div>
<div class="collapsible-body">
<p>
It's hard to write that down (although you could use “”), so we have a symbol especially for it, which is the Greek letter epsilon: {math}\epsilon{math end}.
You'll come across {math}\epsilon{math end} quite often with formal languages.
</p>
</div>
</li>
</ul>
</div>